src/components/Home.jsx

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+import { NavLink } from 'react-router-dom';
+import { useSelector } from 'react-redux';
+
+import continentsImages from '../assets/bd/continentsImages';
+import regionListArray from '../assets/bd/regionList';
+
+import logo from '../assets/img/logo.svg';
+import '../styles/Home.css';
+
+const Home = () => {
```

Comment:
- [ ] As required by the project requirement, the Home page should have a list of categories that could be filtered using given parameters, e.g. category name. 

For Example: You could have a search bar to search for continents by name